### ISSUE TITLE
Document usage and motivation for createAndExercise

### ISSUE THREAD

**Issue by leonidr-da** [digital-asset/daml#10473]
I find the subtleties of using this command a bit difficult to grasp. A lot of times, I think whether I should abandon my pattern of having a Role contract with nonconsuming choices in favor of createAndExercise, but I have not had a clear aha moment.

**Comment by cocreature** [comment]
I think this might make for a nice forum question rather than docs.

**Comment by leonidr-da** [comment]
True, but I do want documentation to be able to point someone at.

**Comment by leonidr-da** [comment]
There is [this](https://discuss.daml.com/t/what-is-a-typical-use-case-for-createandexercise/464) post on the forum but it lacks an actual example, it isn't placed in context. This is not a high priority issue, just a marker to track this request.

**Comment by cocreature** [comment]
I think my problem is that I’m not quite clear where we could put the example such that people actually find it. The API docs for `createAndExercise` are probably not the right place for a discussion on role contracts vs createAndExercise. You could put it somewhere in the daml intro but that’s already long and it’s hard to find specific things in there. So a forum question that ask about role contracts vs createAndExercise seems ilke a nice way to sidestep this issue but if you or someone else has a good idea where this could fit into our docs, I’m also open to adding it there.
